layers/meta-resin-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```
@@ -75,6 +76,7 @@ do_deploy_append() {
     install -m 0600 "${D}/boot/extlinux/extlinux.conf_flasher" "${DEPLOYDIR}"
 
     cp ${WORKDIR}/tegra186-tx2-cti-ASG001-USB3.dtb "${DEPLOYDIR}"
+    cp ${WORKDIR}/tegra186-quill-p3310-1000-c03-00-base.dtb "${DEPLOYDIR}"
```

I guess if we are doing a subdir to distinquish the file from the base jetson.dtb then I also need to move it here?
```suggestion
    cp ${WORKDIR}/tegra186-quill-p3310-1000-c03-00-base.dtb "${DEPLOYDIR}/n510"
```